app.js

```diff
@@ -40,57 +39,61 @@ function verifyProductName(productName) {
 }
 
 /**
- * Verifies a account code
+ * Verifies an account code
  * @param {string} accountCode - The account code to verify
- * @return {boolean} - Whether the account code is verified or not
+ * @return {boolean} - Whether the cost code is verified or not
  */
 function verifyAccountCode(accountCode) {
-  return (accountCode[0]=='S')
+  const accountCodePattern = /^s\d{4}$/i
+  return (accountCodePattern.test(accountCode.trim()))
```

Comment:
Could we validate subcostcodes with it as well? Or add another action to validate subcostcodes?

Reply:
I don't think it is necessary to validate sub- cost codes, at this stage, if the user is never able to enter one as such; it is always provided.